### PR TITLE
feat: veOlas feature to staging

### PR DESCRIPTION
**Pull request by mohandast52**
* Release branch of veOlas containing different features (everything tested locally with metamask)

### DESIGN REVIEW PR
* https://github.com/valory-xyz/autonolas-investor-frontend/pull/27

### VEOLAS
1. Homepage for the veOlas displaying different information

2. Add to lock
  a. User can add the `amount` and `unlock time` to lock
  b. User can select `max` to lock all the amount
  c. Users can't select less than a week & more than 4 years for the unlock time.

3. Increase the amount and unlock time
  a. If a lock already exists, the user can increase the specified amount (Can't lock if no balance is available)
  b. If a lock already exists, the user can increase the unlock time

4. User can also `claim all` if the lock is expired

### BUOLAS

5. Home page for the buOlas displaying different information
  a. User can also `claim all` if the lock is expired

### CODE REVIEW

**Comment by kupermind** [comment]
@mohandast52 in the very first window of creating a lock, if already locked - I would add to the existent phrase: "Amount already locked, please wait until the lock expires, `manage your lock`(link to the manage lock screen), or connect with a different account".

File: components/Home/VeOlas/ManageYourLock/IncreaseUnlockTime.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  );
+  const mappedEndTime = useSelector(
+    (state) => state?.setup?.mappedBalances?.endTime || null,
+  );
+
+  const [form] = Form.useForm();
+
+  const onFinish = async (e) => {
+    try {
+      const txHash = await updateIncreaseUnlockTime({
```
Comment (kupermind) [inline components/Home/VeOlas/ManageYourLock/IncreaseUnlockTime.jsx:32]: Where is this checked for the time to be not longer than the max possible one? It is checked on-chain, but I suppose it's checked off-chain as well?

File: components/Home/VeOlas/ManageYourLock/IncreaseAmount.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    (state) => state?.setup?.mappedBalances?.isMappedAmountZero || false,
+  );
+  const olasBalance = useSelector((state) => state?.setup?.olasBalance);
+  const hasNoOlasBalance = Number(olasBalance || '0') === 0;
+
+  /**
+   * can increase amount only if the mapped amount is zero (ie. no lock exists)
+   * or if the user has some olas tokens.
+   */
+  const cannotIncreaseAmount = isMappedAmountZero || hasNoOlasBalance || !account;
```
Comment (kupermind) [inline components/Home/VeOlas/ManageYourLock/IncreaseAmount.jsx:34]: Not sure I understand this. THe account can increase if there's already a lock in veOLAS, cannot lock if there is no veOLAS locked. Also, is there a check against the input OLAS amount that the account wants to increase the lock for? There's a check on-chain, wondering if there's one off-chain as well.

Comment (kupermind) [reply components/Home/VeOlas/ManageYourLock/IncreaseAmount.jsx:34]: There could be a check for `OLAS.balanceOf(account)` - if it is less than or equal than the input value for the increase amount. This way the user would not be able to input more OLAS to increase the lock for than they actually have. The same could be applied (or it is already applied) to the `createLock()` function.

File: .env.example [hunk 3]
```diff
@@ -1,4 +1,4 @@
 NEXT_PUBLIC_INFURA_ID=__ID__
 NEXT_PUBLIC_MAINNET_URL=__URL__
 NEXT_PUBLIC_GOERLI_URL=__URL__
-NEXT_PUBLIC_AUTONOLAS_URL=__URL__
\ No newline at end of file
+NEXT_PUBLIC_GOVERNANCE_URL=__URL__
```
Comment (oaksprout) [inline .env.example:4]: I've left a bunch of comments about [issues on the experience side](https://docs.google.com/document/d/1IU_KtuOK-dOFCzbbXqYfJOaFobZjvUGeO-rDnbqA1e4/edit#). Want to resolve those before looking at the code.

**Review by oaksprout — APPROVED** [review]

**Pull request merged**